eval/bbh/run_eval.py

```diff
@@ -21,10 +22,16 @@ def eval_hf_model(args, model, tokenizer, examples, task_prompt, save_path=None)
 
     prompts = []
     for example in examples:
+        prompt = task_prompt.strip() + "\n\nQ: " + example["input"]
         if args.use_chat_format:
-            prompt = "<|user|>\n" + task_prompt.strip() + "\n\nQ: " + example["input"] + "\n<|assistant|>\nA:"
+            messages = [{"role": "user", "content": prompt}]
+            prompt = eval(args.chat_formatting_function)(messages, add_bos=False)
```

Comment:
eval + * import seems like not the best way to do this, imo - would it be possible to e.g. import a class/method from a user-specific file (using https://docs.python.org/3/library/importlib.html#importlib.import_module or similar)?

Reply:
Yeah, this might be a better way. Let me change this.